src/languages/autoit.js

```diff
@@ -1,25 +1,46 @@
 
   const COMMENT = {
     variants: [
+      hljs.COMMENT(/^#/, /$/, {
+        relevance: 0
+      }),
```

Comment:
This isn't needed if we're happy with the current behavior.

`
#something
`

Is this a comment or a meta?  Your own highlighting makes it look "meta"... which is the current behavior... changing it to a comment would actually be a bit more complex than just adding this rule.

Reply:
```suggestion
```

Reply:
This is what broke the tests...